src/main/java/mate/academy/spring/model/dto/response/OrderResponseDto.java

```diff
@@ -0,0 +1,45 @@
+package mate.academy.spring.model.dto.response;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import mate.academy.spring.model.Ticket;
+import mate.academy.spring.model.User;
+
+public class OrderResponseDto {
+    private Long id;
+    private List<Ticket> tickets;
+    private LocalDateTime orderDate;
+    private User user;
```

Comment:
Do not share your internal models via DTO
```suggestion
    private Long id;
    private List<Long> ticketIds;
    private LocalDateTime orderDate;
    private Long userId;
```